org.kde.kweather.json

```diff
@@ -0,0 +1,44 @@
+{
+    "id": "org.kde.kweather",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "kweather",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--system-talk-name=org.freedesktop.GeoClue2"
+    ],
+    "separate-locales": false,
+
+    "modules": [
+        {
+            "name": "kweathercore",
+            "config-opts": [ "-DBUILD_TESTING=OFF" ],
+            "buildsystem": "cmake-ninja",
+            "sources": [ { "type": "git", "url": "https://invent.kde.org/libraries/kweathercore.git", "branch": "release/21.05" } ]
```

Comment:
here and below. 

A `git` source needs a specific `commit`, preferably from a tagged release. Not a branch.

Case in point you fixed the build by retriggering it. Which mean the build is not reproducible.

Reply:
ah, alright!